### ISSUE TITLE
How to run

### ISSUE THREAD

**Issue by VareWe** [Debrup-61/Deep_Forests_GA#4]
There are many .m files. Could you please tell me how to run them to get the result?

**Comment by Debrup-61** [comment]
I havent attached all the files here. I have a base file for that to select the options.

**Comment by VareWe** [comment]
Do you mean that running the base file can automatically call other files? So which is the basic file?

**Comment by Debrup-61** [comment]
Yes there's a base file which calls other files automatically and you can
select your choices there. Repo is just meant as a sample of the work, all
the files are not here.

Best,
Debrup
